cmd/occollector/app/collector/processors.go

```diff
@@ -247,7 +248,7 @@ func startProcessor(v *viper.Viper, logger *zap.Logger) (processor.SpanProcessor
 	_ = metricsExporters
 
 	if builder.DebugProcessorEnabled(v) {
-		dbgProc := processor.NewNoopSpanProcessor(logger)
+		dbgProc := processor.NewTraceExporterProcessor(debugexporter.NewDebugTraceDataExporter(logger))
```

Comment:
nit: naming: debugexporter.NewTraceDataExporter

Reply:
starting to think that we could short `TraceDataExporter` to `TraceExporter` (symmetrically for metrics)

Reply:
done.

Reply:
Done second comment.